### ISSUE TITLE
[Feature] Theme for ngx-timepicker-field

### ISSUE THREAD

**Issue by dancUB** [Agranom/ngx-material-timepicker#190]
It would be nice to be able to easily modify the theme for ngx-timepicker-field. Particularly, being able to modify  `$primary-color: deepskyblue` would be useful in making this match a given app's theme.

The theme for the clock is very useful, but to allow for full stylistic compatibility one should be able to modify font color, the underline beneath the field, and the underline on focus (or remove the latter two, if necessary). 
<img width="254" alt="image" src="https://user-images.githubusercontent.com/43146396/63883434-cee7fc80-c999-11e9-810b-2fd0c319bea8.png">

Apologies if this is already possible, but I can't tell that it is.

**Comment by Agranom** [comment]
I agree with you. It's just a first version of `ngx-timepicker-field`. I gonna improve it

**Comment by Jadamae77** [comment]
Is there any way to change the underline color currently?

**Comment by Agranom** [comment]
You can change it by using global styles or using `::ng-deep`

**Comment by Jadamae77** [comment]
Thanks! I am having a little trouble getting the underline changed to the Material purple...what am I missing? My :after element is being placed inside the arrow div, and I can't figure out why.
![image](https://user-images.githubusercontent.com/63246953/78914376-4fa2c700-7a58-11ea-8876-d3e995e4145c.png)
`div.ngx-timepicker-control--active ::after {
    content: '';
    position: absolute;
    bottom: -2px;
    left: 0;
    width: 100%;
    height: 1px;
    background-color: #673ab7;
}`
